test/src/receiver/main.cpp

```diff
@@ -27,38 +39,67 @@ using hclock = chrono::high_resolution_clock;
 using TIME = NDTime;
 
 
-/***** SETING INPUT PORTS FOR COUPLEDs *****/
+/**
+ * Setting input ports for messages
+*/
 struct inp : public cadmium::in_port<Message_t> {};
 
-/***** SETING OUTPUT PORTS FOR COUPLEDs *****/
+/**
+ * Setting output ports or messages

```

Comment:
It would be 'setting output ports for messages'

Reply:
solved